gf-webapp/src/logic/bookingPage/BookingPage.tsx

```diff
@@ -1,17 +1,42 @@
+import {useState} from "react";
 import {PageBorder} from "src/component/pageBorder/PageBorder";
 import {BookingPromo} from "src/logic/bookingPage/bookingPromo/BookingPromo";
-import {HotelPlan} from "src/logic/bookingPage/hotelPlan/HotelPlan";
+import {Filter} from "src/component/filter/Filter";
+import {FilterRoomsContext} from "src/component/filter/FilterContext";
+import {RoomsBlockWithFilters} from "src/component/filter/RoomsBlockWithFilters";
 import styles from "src/logic/bookingPage/BookingPage.module.scss";
 
 export const BookingPage = () => {
+  const [searchValue, setSearchValue] = useState("");
+  const [dateInValue, setDateInValue] = useState("yyyy-MM-dd");
+  const [dateOutValue, setDateOutValue] = useState("yyyy-MM-dd");
+  const [adultsValue, setAdultsValue] = useState(0);
```

Comment:
Why 0?  Magic number

Reply:
Create const MIN_ADULTS_VALUE and MIN_CHILDREN_VALUE